gotestcover_test.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"os"
+	"testing"
+)
+
+func TestParseFlags(t *testing.T) {
+	os.Args = []string{"gotestcover",
+		"-v",
+		"-a",
+		"-x",
+		"-race",
+		"-cpu=4",
+		"-parallel=2",
+		"-run=abc",
+		"-short",
+		"-timeout=15",
+		"-covermode=atomic",
+		"-parallelpackages=2",
+		"-coverprofile=cover.out",
+	}
+
+	err := parseFlags()
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !flagVerbose {
+		t.Errorf("flagVerbose should be set to true")
+	}
+
+	if !flagA {
+		t.Errorf("flagA should be set to true")
+	}
+
+	if !flagX {
+		t.Errorf("flagX should be set to true")
+	}
+
+	if !flagRace {
+		t.Errorf("flagRace should be set to true")
+	}
+
+	if flagCPU != "4" {
+		t.Errorf("flagCPU is not equal to 4, got %s", flagCPU)
+	}
+
+	if flagParallel != "2" {
+		t.Errorf("flagCPU is not equal to 2, got %s", flagParallel)
+	}
+
+	if flagRun != "abc" {
+		t.Errorf("flagRun is not equal to 'abc', got %s", flagRun)
+	}
+
+	if !flagShort {
+		t.Errorf("flagShort should be set to true")
+	}
+
+	if flagTimeout != "15" {
```

Comment:
`15` is not a valid duration.
You should set `15s` or `15m`.

It doesn't throw an error, because I use a string value in my code.
`go test` will throw the error.
I'll see later if it need to be fixed.